packages/java-packages/azure-sdk-archetype/README.md

```diff
@@ -0,0 +1,31 @@
+# Azure SDK Maven Archetype
```

Comment:
@srnagar please put this under the "tools" folder we are trying get all the new tools under there.

Reply:
@weshaggard this archetype is mainly for customers of Azure SDK. This is not a tool that the Azure SDK team uses. Do you still prefer it to be in "tools" directory instead for "sdk/tools".